### DISCUSSION TITLE
ReCon Icons

### DISCUSSION THREAD

**Discussion by johannaweintraub** [twilio-labs/paste#594] [Request: Icon] [answered]
Johanna Weintraub Today at 11:45 AM

Can someone advise me on where the Console engineers can access:

All product icons
twilio branding
bug icon
pinned icon
Logs icon
Insights icon

3 replies

Sarah Li  4 minutes ago
we're in the process of adding those to our repo this sprint

Johanna Weintraub  4 minutes ago
Awesome! @kschwarz please see above and lmk if this will work with your timeframe.

Kate Schwarz  3 minutes ago
Sounds perfect! Thank you!

**Comment by kate-porter** [comment]
Hey @johannaweintraub - do you and your team have what you need on this one now?

**Comment by johannaweintraub** [comment]
Is there a filled pin for when the user pins something from the explore products page?

**Reply by kate-porter** [reply]
Ah, we missed that one since it isn't in the nav itself. I'm chatting with the team now about getting a story into an upcoming sprint to add it. I'll keep you updated!

Are ya good on everything else?

**Comment by johannaweintraub** [comment]
Looks like we still need product icons for

CLI & SDKs
Marketing Campaigns
Connected Devices
Assets
API Explorer

Thanks!

**Reply by kate-porter** [reply]
Since these icons are not available on the Brand website, can you work with Brand to track them down or request that they create them? Once the assets are available, we will get them added to Paste.

**Reply by kate-porter** [reply]
Regarding CLI & SDKs, I do see individual icons for those on the Brand website - are we planning to merge those into one? If so, Brand will need to create a new icon for us (or tell us which one of the existing icons to use). If you don't mind working with them on that, that'd be great.

**Reply by serifluous** [reply]
Hey @johannaweintraub ! We've just added CLI, SDKs, Marketing Campaigns, Connected Devices, Assets, and API Explorer icons to Paste. https://paste.twilio.design/icons

**Comment by johannaweintraub** [comment]
Good question - I will get back to you on that.

**Reply by kate-porter** [reply]
Perfect, thank you!

**Comment by johannaweintraub** [comment]
Looks like CLI and SDKs can remain separate. We just need to add them as
they are to Paste.
Thanks!

**Reply by kate-porter** [reply]
Cool, we'll get those added when we add the filled pin icon. For the remaining icons (Marketing Campaigns, Connected Devices, Assets, API Explorer), if you can work with Brand to get those, that would be great. Then you can share them with us and we'll get them added.

**Comment by johannaweintraub** [comment]
Yay, thank you!

**Reply by kate-porter** [reply]
Sure thing! Do you mind if I close this discussion out for now, and when you have the new icons from Brand, we can create a new discussion?

**Comment by johannaweintraub** [comment, marked as answer]
Sure.

**Reply by kate-porter** [reply]
Cool. We're getting the filled pin, CLI, and SDK icons created in Sketch this sprint and added to Paste next sprint, and we've got a note to followup with you and team once they're added. Thank you!
